docs/book/src/getting-started/testdata/project/Makefile

```diff
@@ -152,10 +152,10 @@ $(LOCALBIN):
 
 ## Tool Binaries
 KUBECTL ?= kubectl
-KUSTOMIZE ?= $(LOCALBIN)/kustomize-$(KUSTOMIZE_VERSION)
-CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen-$(CONTROLLER_TOOLS_VERSION)
-ENVTEST ?= $(LOCALBIN)/setup-envtest-$(ENVTEST_VERSION)
-GOLANGCI_LINT = $(LOCALBIN)/golangci-lint-$(GOLANGCI_LINT_VERSION)
+KUSTOMIZE ?= $(LOCALBIN)/kustomize
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+ENVTEST ?= $(LOCALBIN)/setup-envtest
+GOLANGCI_LINT = $(LOCALBIN)/golangci-lint
```

Comment:
What happens if I bump 3 diff versions?
Will we keep the old 3 versions or we deleted/rewrite then?

Reply:
But what is the result?
How we would have 3 bin/kustomize with the version in the name?

Reply:
like this, maybe I misunderstood the question?
![image](https://github.com/user-attachments/assets/44f3a2d0-e1d1-4b01-8507-e343f23d1989)

Reply:
Yes, but the kustomize-v5.4.0 or   kustomize-v5.4.1 are the binaries downloaded once right?
And kustomize is the symlink. Am I right?

Reply:
`kustomize` is the symlink (as IntelliJ shows with that little arrow).
The symlink is always recreated, regardless whether the version already exists or not, so it will always point to the version in the makefile. (this was changed with the last push)

Reply:
Thank you for confirm my understand.